consumers/hidden-test-consumer/src/hidden_test_consumer/process.py

```
@@ -279,7 +315,7 @@ def process(self) -> Iterator[ProcessRequest]:
         for step in steps:
             try:
                 yield from step()
-                sleep(2)
+                sleep(randint(1, 3))
```

Ruff (0.8.2)

Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)